src/components/Form.js

```diff
@@ -74,42 +105,46 @@ export default class Form extends Component {
     return null;
   }
 
-  onChange = (formData, options={validate: false}) => {
+  onChange = (formData, options = {validate: false}) => {
```

Comment:
Of course, I have some problems when switching between projects, but the current code was lintener test, so maybe we need some fixes in .eslinrc?